.drone.yml

```diff
@@ -176,7 +176,6 @@ pipeline:
   report-coverage:
     image: plugins/coverage
     server: https://coverage.vmware.run
-    token: ${GITHUB_AUTOMATION_API_KEY}
```

Comment:
I think we need this to avoid hitting the non-token limit, which is very strict?